reference/pipeline/stages/index.md

```diff
@@ -13,10 +13,27 @@ sidebar:
 This article describes the currently-supported stages that you can add to your
 Spinnaker pipelines.
 
+Note that when you're creating a pipeline, you probably won't see every stage
+that's listed here: you'll only see the stages that Spinnaker supports on your
+provider.
+
 ## General
 
 ### Bake
-Bake an image in the specified region.
+Bake an image from the specified packages. _Baking_ is the process of
+creating a machine image. Spinnaker's bakery is backed by
+[Hashicorp's Packer](https://www.packer.io/intro/). Spinnaker provides default
+[Packer templates](https://www.packer.io/docs/templates/index.html) and base
+machine images in order to get you started, but see the
+[bakery configuration guide](/setup/bakery/) if you want to customize your bake
+process.
+
+Note that Spinnaker skips the bake process if it detects that a new bake is
+unnecessary. Spinnaker generates a unique key for each bake based on the bake
+stage parameters: base OS, versioned packages, etc. If either the packages or
+the bake stage parameters have changed, Spinnaker should trigger a new bake.
```

Comment:
s/should trigger/triggers/

Unless, @lwander , is there some doubt that the new bake'll get triggered upon a change?